### ISSUE TITLE
Add copyright headers to tests

### ISSUE THREAD

**Issue by 1st1** [PyO3/tokio#28]
```
Copied from the uvloop project.  If you add a new unittest here,
please consider contributing it to the uvloop project.

Portions copyright (c) 2015-present MagicStack Inc.  http://magic.io
```

**Comment by fafhrd91** [comment]
Should we move tests to separate package?

**Comment by 1st1** [comment]
Yes, that would be an ideal solution. Let me try to do that for uvloop project first.

**Comment by fafhrd91** [comment]
also let's convert tests to proper pytest style. we can use what I did for tokio.

**Comment by 1st1** [comment]
> also let's convert tests to proper pytest style. we can use what I did for tokio.

I'd like to keep compatibility with unittest. `pytest` still has weird policy about showing warnings etc., I don't trust it :(

**Comment by fafhrd91** [comment]
Ok. Pytest tests just looks so much better :)

**Comment by 1st1** [comment]
Hard to disagree :) Anyways, I'll spend some time to re-evaluate it.
